src/App.css

```diff
@@ -121,14 +127,28 @@ h1, h2, h3, h4, h5, h6, span {
   right: 0;
   display: flex;
   justify-content: flex-end;
-  width: 50%
+  width: 50%;
 }
 
 #welcome-and-logout > * {
   font-family: "Lora", serif;
   margin: 0 20px;
 }
 
+#points-display {
+  height: 50px;
+  width: 50px;
+  background-color: #e8b704;
+  border-radius: 50%;
+  margin: -30px 0;
+}
+#karma-points-amount {
+  color: #88a65e;
```

Comment:
Even though this is what we have in our Lo-Fis, I think we maybe should go with white instead? The green on yellow isn't very readable.